src/Particle/DistanceTableData.h

```diff
@@ -312,6 +312,12 @@ struct DistanceTableData
    * @param P the target particle set
    */
   virtual void evaluate(ParticleSet& P) = 0;
+  virtual void mw_evaluate(const RefVector<DistanceTableData>& dt_list, const RefVector<ParticleSet>& p_list)
+  {
+    #pragma omp parallel for
```

Comment:
I don't think any bare omp pragmas outside of offload kernels/setup should be entering the code base. 
They are doubly counter intuitive in `mw_` methods.  That prefix was for calling blocked evaluations from multiple walker (mw) per cpu thread task contexts (i.e. crowds). So I'd expect this only to be called from unified driver code and all available threads should already be in use.  

The reason for this must be documented and if it is just to support the legacy drivers or some difficiency in the open mp offload implementation it documented along with the conditions that would allow it to be deleted.

Reply:
It should not be counter intuitive. All the mw_ interfaces should execute walkers on a GPU or a team of threads which is below the level of Crowds. Consider AMD epic processor with 64 cores, even if there can be 4 GPU per socket, 16 crowds per GPU can be too much. In the current setting, 4 crowds each using 4 threads can be enabled to balance GPU and CPU cores and max out throughput.